packages/babel-preset-typescript/test/fixtures/jsx-compat/tsx-valid/output.js

```diff
@@ -1 +1,6 @@
-<div />;
```

Comment:
This output shouldn't change because we only apply `syntaxJSX` in this example and the JSX syntax should be parsed without transformed.

Reply:
@JLHwung Good catch! Just fixed it.